src/Botty.ts

```diff
@@ -18,11 +18,14 @@ export default class Botty {
     public readonly client = new Discord.Client();
     private personalSettings: PersonalSettings;
     private sharedSettings: SharedSettings;
+    private version: string;
 
     constructor(sharedSettings: SharedSettings) {
         this.personalSettings = sharedSettings.botty;
         this.sharedSettings = sharedSettings;
         console.log("Successfully loaded bot settings.");
+        
+        require("fs").readFile("version", "utf-8", (err,data) => this.version = data);
```

Comment:
```
this.version = fs.readFileSync("version", "utf8");
```
Probably cleaner to import fs instead of doing a require. Typescript types an `import` properly, while it types `require` as any.